frontend/src/css/navbar.css

```diff
@@ -15,14 +17,25 @@
 }
 
 .nav-container {
-  width: 20%;
+  position: relative;
+  width: 15%;
   height: 100%;
-  margin-top: 1%;
+  margin-top: 0.7%;
```

Comment:
I suppose that by putting the margins as percentages when the window size get smaller the components also move "upwards" when they should only move to the sides.